### PR TITLE
[DELETEME] Add MacOS support and include system info in code system calibration

### PR DESCRIPTION
**Pull request by DavidVeksler**
* Improves OS detection
* Provides additional system/environment information to improve commands

### CODE REVIEW

**Review by AzureDominus — COMMENTED** [review]
Is all of this extra information really needed though? It adds tokens to the request which makes the call a bit more expensive. Maybe only the python version and platform are needed to add a bit of context.

File: engshell.py [hunk 1]
```diff
@@ -61,112 +77,128 @@ def LLM(prompt, mode='text'):
...
+    logging.debug(response_content)
     return response_content
 
+
 def containerize_code(code_string):
     code_string = code_string.replace('your_openai_api_key_here', OPENAI_KEY)
     # uncomment this if you wish to easily use photos from Unsplash API
-    # code_string = code_string.replace('your_unsplash_access_key_here', UNSPLASH_ACCESS_KEY)
+    code_string = code_string.replace(
+        'your_unsplash_access_key_here', UNSPLASH_ACCESS_KEY)
```
Comment (emcf) [inline engshell.py:108]: We want to keep unsplash API features off by default -- they make engshell harder to use and only add value if you are doing tasks with stock photos on the fly

File: prompts.py [hunk 2]
```diff
@@ -1,28 +1,40 @@
 from keys import OPENAI_KEY
 import platform
+import system_info
 
-WINDOWS = platform.system() == "Windows"
-OPERATING_SYSTEM = "windows" if WINDOWS else "linux"
+
+SYSTEM_INFO = system_info.get_system_info()
```
Comment (emcf) [inline prompts.py:6]: System info is overkill, just need OS

**Comment by emcf** [comment]
Other changes look great!

File: engshell.py (hunk 1 above)
Comment (DavidVeksler) [reply engshell.py:108]: Sorry, did not mean to push this to you.  I'm new to GitHub collab features.  You can delete this PR.

**Pull request closed without merging**